modules/ipi-install-retrieving-the-openshift-installer.adoc

```diff
@@ -14,3 +14,8 @@ available version of {product-title}:
 $ export VERSION=latest-4.7
 export RELEASE_IMAGE=$(curl -s https://mirror.openshift.com/pub/openshift-v4/clients/ocp/$VERSION/release.txt | grep 'Pull From: quay.io' | awk -F ' ' '{print $3}')
 ----
+
+[NOTE]
+====
+The recommended update channels for the OpenShift Container Platform are the `stable` or `fast` channels. The `fast` release channel is the latest version of the OpenShift Container Platform, published as soon as the version is declared a general availability release. This version is a fully supported, production quality build that has succeeded the candidate build. The `stable` release channel provides a consistent experience by only adding new releases from the `fast` channel after a period of evaluation by Red Hat support teams.
```

Comment:
- s/the OpenShift Container Platform/{product-title}
- Is the release channel==the version? This sentence sort of conflates the two concepts: "The `fast` release channel is the latest version of the OpenShift Container Platform, published as soon as the version is declared a general availability release."
- Possibly consider bulleting the two channel descriptions?
- The reporter for this bug asked about adding this definition "Wherever such statements are made" - do you know if Vikram intended you to do that or just to address this instance for this bug effort?

Reply:
Hey @jeana-redhat, 

I updated the ticket a bit to be an xref to the original topic that describes these channels. I've had it confirmed by the reporter that it is only this instance that they were referring to. Can you review and merge if approved?